.run/Extract.run.xml

```diff
@@ -6,8 +6,6 @@
     </option>
     <option name="filePath" value="$PROJECT_DIR$/packages/snippets/bin/extract.dart" />
     <option name="workingDirectory" value="$PROJECT_DIR$/../flutter" />
-    <method v="2">
-      <option name="ToolBeforeRunTask" enabled="true" actionId="Tool_External Tools_Remove API" />
-    </method>
+    <method v="2" />
   </configuration>
 </component>
```

Comment:
I'm not familiar with these .run files. What are they configuring? Also should they end with a new line?

Reply:
They're for IntelliJ, so you can share run configurations. They don't have to end with a newline (that's how IntelliJ writes them).